devices/common_patches/f730_f7x2_f7x3_tim.yaml

```diff
@@ -0,0 +1,39 @@
+TIM*:
+  _delete:
+    - CCR6
```

Comment:
We're using CCR%s for CCR[1..4], and not for CCR5 because it's using some more fields. Is there a way to modify the `dim` and `dimIndex` values of the SVD so we can add index 6?
![image](https://user-images.githubusercontent.com/159235/131805492-6ebd3282-fdc9-42b9-9ee3-402de6984c7c.png)
Otherwise I'll revert and fix this one also.

Reply:
I don't think it would be possible for the array to include 5 and 6 because they are not contiguous in memory. It is possible to tell `_array` which registers to use, and currently in `devices/common_patches/tim/tim_ccr.yaml` it says `_array: "CCR[1234]": {}` which should only select the right ones.

I think probably the best we can do is leave CCR5/6 alone as separate registers?

Reply:
Addressed in 545e6ff9